### PR TITLE
Update installation on WSL documentation

### PR DESCRIPTION
**Pull request by jeannejoachim**
During the Lethe installation at my industrial partner, we ran into some issues that were not documented in the first version of the page. This PR fixes everything that we saw.

### CODE REVIEW

**Review by bibeauv — APPROVED** [review]
Really nice :)
The tables are really clear!

**Review by blaisb — APPROVED** [review]

**Comment by blaisb** [comment]
Just a small conflict to fix with the latest merge from @lpsaavedra

**Comment by jeannejoachim** [comment]
> Just a small conflict to fix with the latest merge from @lpsaavedra

It's fixed! I really like this new copy-button :)

File: doc/source/installation/windows_wsl.rst [hunk 1]
```diff
@@ -258,18 +308,81 @@ If you have already installed deal.II and lethe, you can update them without doi
...
+.. admonition:: Example for the candi.cfg
+
+	.. code-block:: text
+
+		cd /home/<user_name>/Software/candi
+		nano candi.cfg
+
+2. Modify the text in the file, using only the keyboard. 
+
+.. tip::
```
Comment (voferreira) [inline doc/source/installation/windows_wsl.rst:377]: I'd change it for a note and put it right after line 359

**Review by voferreira — APPROVED** [review]
All good for me. I have a tiny comment, but nothing special. Good one!

File: doc/source/installation/windows_wsl.rst (hunk 1 above)
Comment (blaisb) [reply doc/source/installation/windows_wsl.rst:377]: @jeannejoachim I don't have a strong opinion on this so I'll let you judge. Just tell me when you it's ready for final merge.

Comment (jeannejoachim) [reply doc/source/installation/windows_wsl.rst]: Yes that's a good idea, thank you Victor :)

**Comment by jeannejoachim** [comment]
@blaisb Good to go!

**Pull request merged**